### DISCUSSION TITLE
Uncaught TypeError: import_nprogress.default.configure is not a function #87

### DISCUSSION THREAD

**Discussion by riasvdv** [livewire/livewire#6443] [7. Bugs (archived)]
### Livewire version

v3.0.0-beta.10

### Which PHP version are you using?

PHP 8.2

### Repository URL

https://github.com/riasvdv/livewire-issue

### Steps To Reproduce

1. Add the following script to a livewire page:

```
<script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/monaco-editor/0.21.2/min/vs/loader.min.js"></script>
 ```

Seems like it is some kind of conflict between Livewire/Alpine & the Monaco editor

### Additional information

Reported here before on the livewire/next repo: https://github.com/livewire/next/issues/87

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by robsontenorio** [comment]
@riasvdv  Have you solved it?

**Reply by riasvdv** [reply]
We stopped using Monaco 😅

**Comment by SimonErich** [comment]
Having the same issue!

**Comment by yaovidiy** [comment]
Not sure if it help any one but in my case adding `defer` helped to resolve this problem
